packages/Button/src/Button.tsx

```diff
@@ -407,7 +405,7 @@ export default class Button extends Component<ButtonProps> {
     return (
       <button
         type="button"
-        {...rest as HTMLElementProps<'button'>}
+        {...(rest as HTMLElementProps<'button'>)} //eslint-disable-line
```

Comment:
Not seeing this disable being required locally. Hoping we can remove.

Reply:
👍